public/app/features/scenes/apps/traffic.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+
+import {
+  SceneFlexLayout,
+  SceneTimePicker,
+  VizPanel,
+  EmbeddedScene,
+  SceneTimeRange,
+  VariableValueSelectors,
+  SceneControlsSpacer,
+  SceneRefreshPicker,
+  SceneFlexItem,
+  SceneObjectState,
+  SceneObjectBase,
+  SceneObjectUrlSyncConfig,
+  SceneObjectUrlValues,
+} from '@grafana/scenes';
+import { Button } from '@grafana/ui';
+
+import { getInstantQuery, getTimeSeriesQuery, getVariablesDefinitions } from './utils';
+
+export function getTrafficScene(): EmbeddedScene {
+  const httpHandlersTable = new VizPanel({
+    $data: getInstantQuery({
+      expr: 'sort_desc(avg without(job, instance) (rate(grafana_http_request_duration_seconds_sum[$__rate_interval]) * 1e3)) ',
+    }),
+    pluginId: 'table',
+    title: 'Handlers',
+    options: {
+      footer: {
+        enablePagination: true,
+      },
+    },
+    fieldConfig: {
+      defaults: {},
+      overrides: [
+        {
+          matcher: {
+            id: 'byRegexp',
+            options: '.*',
+          },
+          properties: [{ id: 'filterable', value: false }],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Time',
+          },
+          properties: [{ id: 'custom.hidden', value: true }],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value',
+          },
+          properties: [{ id: 'displayName', value: 'Duration (Avg)' }],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'handler',
+          },
+          properties: [
+            {
+              id: 'links',
+              value: [
+                {
+                  title: 'Go to handler drilldown view',
+                  url: '/scenes/grafana-monitoring/traffic?handler=${__value.text:percentencode}',
+                  // onBuildUrl: () => {
+                  //   const params = locationService.getSearchObject();
+                  //   return getLinkUrlWithAppUrlState('/scenes/grafana-monitoring/traffic', {
+                  //     ...params,
+                  //     handler: '${__value.text:percentencode}',
+                  //   });
+                  // },
```

Comment:
```suggestion
```

Reply:
@dprokop  We really need to expose more ways to render links with preserved url keys. I was not able to build a url here with a new URL query generated from a data link. Will create an issue to describe this issue